import-export-cli/cmd/addApi.go

```diff
@@ -71,7 +72,12 @@ func handleAddApi(nameSuffix string) {
 	var javaInterceptorsCmNames []string
 
 	for i, flagSwaggerFilePath := range flagSwaggerFilePaths {
-		fmt.Println(fmt.Sprintf("Processing swagger %v: %v", i+1, flagSwaggerFilePath))
+		// log processing only if there are more projects
+		if len(flagSwaggerFilePaths) > 1 {
+			fmt.Println(fmt.Sprintf("Processing swagger %v: %v", i+1, flagSwaggerFilePath))
```

Comment:
Is "Processing swagger .." goes for standard output.. usually it's better to use std.err for general info/error messages. Otherwise, if your command gives any output, it's difficult for a shell script etc to filter out those and extract the real output. For this, we can use `utils.Logln`

Reply:
+1. Let's use this util.